org.civicrm.bootstrapcivicrm/scss/civicrm/absences/_absences.scss

```diff
@@ -0,0 +1,58 @@
+.page-civicrm-absence-period,
```

Comment:
You are both amending _hrabsence.css_ and creating a new partial under _org.civicrm.bootstrapcivicrm/scs/civicrm/_. What's the reasoning here? Couldn't it be either one or the other? Why both?